spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/micrometer/RabbitListenerObservation.java

```diff
@@ -24,19 +24,16 @@
 
 /**
  * Spring Rabbit Observation for listeners.
- *
```

Comment:
Please, consider to revert all the black line changes.
This is really not OK to have the code like this when you read it.
Thanks

Reply:
Do you have a config file to use for intelliJ or only the eclipse one ? it would be much more easier since it doesn't like some of the blank lines you're mentioning

Reply:
I use this: https://github.com/spring-projects/spring-integration/blob/main/src/idea/spring-framework.xml.

Will copy it over here into this project later.

Thanks for the pointer!

Reply:
Thanks for the link, I'll use for further contributions so I don't mess things around...